### PR TITLE
Remove index_dev.php

### PR DESCRIPTION
**Pull request by mabumusa1**
The title says it all

### CODE REVIEW

File: .ddev/local.config.php.dist [hunk 1]
```diff
@@ -52,10 +52,10 @@ return [
     'version'           => 'OAuth1a',
 
     // Required for OAuth1a and OAuth2
-    'baseUrl'           => 'http://localhost/mautic/index_dev.php',
+    'baseUrl'           => 'http://localhost/mautic/index.php',
```
Comment (mollux) [inline .ddev/local.config.php.dist:55]: @mabumusa1 why specify `/index.php` and not `/`?
I see no benefit doing so (here and in all other files

Comment (mabumusa1) [reply .ddev/local.config.php.dist]: @mollux There is no benefit, I just removed them. Thanks for the feedback on this, if it is good let's merge it

Comment (mabumusa1) [reply .ddev/local.config.php.dist:55]: @mollux I just checked the workflow settings again, we do not have `mod_rewrite` enabled, so we need to explicitly specify `index.php`

Comment (escopecz) [reply .ddev/local.config.php.dist:55]: @mabumusa1 can you push?

Comment (mabumusa1) [reply .ddev/local.config.php.dist:55]: @escopecz I just did, when CI is green you can merge

**Pull request merged**